sr_hand/launch/rviz/rviz_motor.launch

```diff
@@ -1,6 +1,18 @@
 <launch>
   <node pkg="rviz" name="rviz_visualizer" type="rviz" />
-  <include file="$(find sr_hand)/launch/load_hand_model.launch" />
```

Comment:
@guihomework can you remove those changes from this pull request? They aren't really related.